app/uk/gov/hmrc/gform/LookupLoader.scala

```diff
@@ -17,29 +17,39 @@
 package uk.gov.hmrc.gform
 
 import com.miguelfonseca.completely.AutocompleteEngine
-import com.miguelfonseca.completely.text.analyze.tokenize.WordTokenizer
 import com.miguelfonseca.completely.text.analyze.transform.LowerCaseTransformer
 import kantan.csv._
 import kantan.csv.ops._
 import uk.gov.hmrc.gform.sharedmodel.LangADT
 import uk.gov.hmrc.gform.sharedmodel.formtemplate.Register
-import uk.gov.hmrc.gform.lookup.{ AjaxLookup, LocalisedLookupOptions, LookupAdapter, LookupId, LookupInfo, LookupLabel, LookupOptions, LookupRecord, LookupType, RadioLookup, ShowAll }
+import uk.gov.hmrc.gform.lookup._
 
 class LookupLoader {
 
-  implicit private val lookupIdCellDecoder: CellDecoder[LookupId] = implicitly[CellDecoder[String]].map(LookupId)
+  implicit private val lookupIdCellDecoder: CellDecoder[LookupId] =
+    implicitly[CellDecoder[String]].map(s => LookupId(s.trim))
   implicit private val lookupLabelCellDecoder: CellDecoder[LookupLabel] =
     implicitly[CellDecoder[String]].map(s => LookupLabel(s.trim))
+  implicit private val lookupPriorityCellDecoder: CellDecoder[LookupPriority] =
+    implicitly[CellDecoder[Option[Int]]].map(p => LookupPriority(p.getOrElse(1)))
```

Comment:
Instead of hardcoding 1 in many places, you could created a constant `DEFAULT = LookupPriority(1)`